extensions/azure/cosmos/cosmos-common/src/main/java/org/eclipse/dataspaceconnector/azure/cosmos/CosmosDbApiImpl.java

```diff
@@ -130,8 +131,9 @@ public void deleteItem(String id) {
         try {
             container.deleteItem(item, itemRequestOptions).getItem();
```

Comment:
I think the `getItem` call it's not needed

Reply:
Good catch. I removed it.